browsepy/plugin/gallery/static/css/gallery.css

```diff
@@ -0,0 +1,171 @@
+body {
```

Comment:
There is a lot of redundant stuff here, and some general browsepy CSS rules should not have been overriden.